src/main.rs

```diff
@@ -55,6 +55,7 @@ Options:
     -f --render <file>  Render a specific markdown file
     -o --os <type>      Override the operating system [linux, osx, sunos]
     -u --update         Update the local cache
+    -a --auto-update    Update the local cache automatically if out of date
```

Comment:
Hmm, "if out of date" might indicate (wrongly) that the cache is updated if the upstream changes.

Maybe a wording like this would be better?

```suggestion
    -a --auto-update    Update the local cache automatically if older than 30 days
```

The `30 days` part might be wrong if we change `MAX_CACHE_AGE` but I'd prefer to keep the usage string static.

Reply:
> The 30 days part might be wrong if we change MAX_CACHE_AGE but I'd prefer to keep the usage string static.

I choose my wording exactly because of that :stuck_out_tongue_closed_eyes: but yes, for the user this message is probably less confusing.